### ISSUE TITLE
 Does not zoom in from the center

### ISSUE THREAD

**Issue by bariskrdrl** [timmywil/panzoom#482]
### Describe the bug

I zoom in on an image with a resolution of 11000px. When I use the zoom in and zoom out functions, I cannot zoom in from the center. When I try a 1600px image, it zooms from the center.

### Your environment

- Version of panzoom: 4.1.0
- Browser and browser version: Google Chrome | 81.0.4044.122 (Official Compilation) (64-bit)

### Expected behavior

Center-focused zoom in and zoom out should be

### Actual behavior

When I do zoom in, the right works down.

**Comment by timmywil** [comment]
Thanks for opening an issue. There's a difference between zooming from the center of the viewport and the center of the image. The transform-origin described [here](https://github.com/timmywil/panzoom#faq) is what controls centering for HTML, and this is based on the panzoom element (or image). To center on the viewport when the image is not centered, you'll need to calculate the center of the viewport, and then use the [zoomToPoint](https://github.com/timmywil/panzoom#zoomtopoint) method.

In other words, I think the default should stay the way it is. And the tools are there to center the way you're describing, it just doesn't do it automatically.

If you make an example on https://jsbin.com, we can iron it out further.
